doc/files/package.json.md

```diff
@@ -393,6 +393,15 @@ shortcut syntax you use for `npm install`:
 
     "repository": "gitlab:user/repo"
 
+If your code is part of a monorepo, you should specify the directory in which
+the `package.json` for this package lives:
+
+    "repository": {
+      "type" : "git",
+      "url" : "https://github.com/facebook/react.git",
+      "directory": "packages/react-dom"
```

Comment:
Should this be `./packages/react-dom` to show its a relative path to the root of the repo?

Reply:
I'm fine with / without the `./` prefix. I think it's relatively clear that this would always be relative to the root of the repo.

Reply:
Ok let's leave it as is. I just submitted a PR to use your syntax here. https://github.com/zeit/now-builders/pull/185

Reply:
I would expect both to work but prefer to use the syntax without `./` because that's what you usually do when referring to files with the `git` CLI and how the git CLI displays them (e.g. `git status`, `git log -p` etc).